docs/configuration/configuration.md

```diff
@@ -2471,9 +2471,9 @@ anchored on both ends. To un-anchor the regex, use `.*<regex>.*`.
 * `keep`: Drop targets for which `regex` does not match the concatenated `source_labels`.
 * `drop`: Drop targets for which `regex` matches the concatenated `source_labels`.
 * `hashmod`: Set `target_label` to the `modulus` of a hash of the concatenated `source_labels`.
-* `labelmap`: Match `regex` against all label names. Then copy the values of the matching labels
-   to label names given by `replacement` with match group references
-  (`${1}`, `${2}`, ...) in `replacement` substituted by their value.
+* `labelmap`: Ignore `source_labels` and match `regex` against all source label names. Then
```

Comment:
IMO this is still a little bit confusing and could imply that regex is being matched against all source labels _except_ (ignoring) the `source_labels`.

```suggestion
* `labelmap`: Match `regex` against all source label names, not just those specified in `source_labels`. Then
```

Maybe something like this is better?